R/TealAppDriver.R

```diff
@@ -65,6 +65,21 @@ TealAppDriver <- R6::R6Class( # nolint: object_name.
         )
       )
 
+      # Check for minimum version of Chrome that supports the tests
+      #  - Element.checkVisibility was added on 105
+      chrome_version <- numeric_version(
+        gsub(
+          "[[:alnum:]_]+/", # Prefix that ends with forward slash
+          "",
+          self$get_chromote_session()$Browser$getVersion()$product
+        ),
+        strict = FALSE
+      )
+
+      testthat::skip_if(is.na(chrome_version), "Problem getting Chrome version.")
+      testthat::skip_if(chrome_version < 105, "Chrome version is not supported, please upgrade.")
```

Comment:
please include `chrome_version` in the err message - it would help debugging

Reply:
Oh, great idea!

Error message in the case required version is 125 (to force it, instead of 105 that we're targetting)

```r
Error: Reason: Chrome version '124.0.6367.60' is not supported, please upgrade to '125' or higher
```